notification-support/src/test/java/com/expediagroup/streamplatform/streamregistry/core/events/CustomSchemaMethodsLoadingTest.java

```diff
@@ -100,10 +100,18 @@ public void having_defined_custom_parser_methods_verify_they_execute_properly()
   }
 
   public static AvroKey myCustomKey(Schema schema) {
-    val key = AvroKey.newBuilder()
-        .setId(schema.getKey().getName())
-        .setVersion(null)
+    val name = schema.getKey().getName();
```

Comment:
```suggestion
    var key = AvroKey.newBuilder()
        .setId(schema.getKey().getName())
        .setParent(AvroKey.newBuilder()
              .setId(schema.getKey().getDomain())
              .setType(AvroKeyType.DOMAIN)
              .build())
        .setType(AvroKeyType.SCHEMA)
        .build();
```
I would consider nesting as above, but I guess it is personal preference.